yarn.lock

```diff
@@ -3638,7 +3638,7 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4, create-hmac@^1.1.7:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-cross-fetch@^3.0.6:
+cross-fetch@^3.0.6, cross-fetch@^3.1.4:
```

Comment:
maybe we could also bump cross-fetch version in @trezor/rollout so we don't need two versions here?

Reply:
cool idea.